### PR TITLE
Gitops-1311 added support matrix to readme

### LINKED ISSUE #271: OCP Support Matrix for kam
**Describe the bug**

There is no specific OCP support matrix for kam found in the README.md. It would be great if we add the OCP support matrix for user reference.

**To Reproduce**
1. Go to 'https://github.com/redhat-developer/kam/tree/master/docs/journey/day1#day-1-operations'
2. No reference for OCP support matrix found

**Expected behavior**

OCP support matrix for user reference

### PR DESCRIPTION
**Pull request by ciiay**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

/kind documentation

**What does this PR do / why we need it**:
We should provide this Support Matrix information to users.

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #271 

**How to test changes / Special notes to the reviewer**:
Check redhat-developer/kam repository and observe Support Matrix exists.

### CODE REVIEW

**Comment by ciiay** [comment]
It looks like this.
Screen Shot 2021-09-22 at 10 10 16 AM

**Comment by ciiay** [comment]
This is how it looks like.
Screen Shot 2021-09-22 at 10 18 25 AM

**Review by ishitasequeira — DISMISSED** [review]
/lgtm
@chetan-rns PTAL

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
-The GitOps Application Manager provides command line interface to bootstrap GitOps and perform other operations.  See why and how [GitOps for Application Delivery](./docs/README.md) can help your business streamline your application delivery.
+The GitOps Application Manager provides command line interface to bootstrap GitOps and perform other operations. See why and how [GitOps for Application Delivery](./docs/README.md) can help your business streamline your application delivery.
+
+## Support Matrix
+
+|                                  KAM                                   | OpenShift | OpenShift GitOps | OpenShift Pipelines |
+| :--------------------------------------------------------------------: | :-------: | :--------------: | :-----------------: |
+|                           0.0.40 (Oct 14th)                            |    4.9    |       1.3        |  1.6.x (Oct 20th)   |
+| [0.0.39](https://github.com/redhat-developer/kam/releases/tag/v0.0.39) |    4.8    |      1.2.1       |        1.5.x        |
+| [0.0.38](https://github.com/redhat-developer/kam/releases/tag/v0.0.38) |    4.7    |      1.2.1       |        1.4.x        |
```
Comment (wtam2018) [inline README.md:11]:
```suggestion
| [0.0.38](https://github.com/redhat-developer/kam/releases/tag/v0.0.38) |    4.7    |      1.2.0      |        1.4.x        |
```

**Pull request merged**